l3kernel/l3box.dtx

```diff
@@ -1727,6 +1742,61 @@
 %    \end{macrocode}
 % \end{macro}
 %
+% \subsection{Vertical boxes to width}
+%
+% \begin{macro}
+%   {
+%     \vbox_set_to_wd:Nnn,  \vbox_set_to_wd:cnn,
+%     \vbox_gset_to_wd:Nnn, \vbox_gset_to_wd:cnn,
+%     \vbox_set_top_to_wd:Nnn,  \vbox_set_top_to_wd:cnn,
+%     \vbox_gset_top_to_wd:Nnn, \vbox_gset_top_to_wd:cnn,
+%   }
+%   We both set the \tn{hsize} and the box width: this covers the case where
+%   the box is empty, for example.
+%    \begin{macrocode}
+\cs_new_protected:Npn \vbox_set_to_wd:Nnn #1#2#3
+  {
+    \tex_setbox:D #1 \tex_vbox:D
+      {
+        \tex_hsize:D \@@_dim_eval:n {#2}
+        \color_group_begin: #3 \par \color_group_end:
+      }
+    \box_dp:N #1 \@@_dim_eval:n {#2}
```

Comment:
do you mean `box_dp` here not `box_wd` ?

Reply:
Ooops :)